reference/catalog.v3.yml

```diff
@@ -6893,6 +6893,9 @@ paths:
 
         **Read-Only Fields**
         * `id`
+
+        **Notes**
+        It takes two separate requests to create a new checkbox modifier with option values. First, perform a request to create a modifier. Then perform a second request to update option values. 
```

Comment:
I just have a slight suggestion that you can take or leave. 😺 

"Then, perform a second request to update option values." or "Perform a request to create a modifier, then perform a second request to update option values."

Reply:
Thank you @bc-tgomez 
Your suggestion is better. I should not have a first without a second.